{
  "metadata": {
    "timestamp": "2025-02-25T08:14:55Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2012-1150",
        "description": "Python before 2.6.8, 2.7.x before 2.7.3, 3.x before 3.1.5, and 3.2.x before 3.2.3 computes hash values without restricting the ability to trigger hash collisions predictably, which allows context-dependent attackers to cause a denial of service (CPU consumption) via crafted input to an application that maintains a hash table.",
        "metadata": {
          "published_date": "2012-10-05T21:55:01.113",
          "last_modified_date": "2024-11-21T01:36:32.533",
          "confidence_level": "low",
          "severity": "medium"
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "poc",
          "automatable": false
        },
        "counts": {
          "public_exploit_count": 1
        },
        "epss": {
          "score": 0.0097,
          "percentile": 0.83489
        },
        "metrics": [
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "2.0",
            "vector_string": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
            "attack_vector": "NETWORK",
            "base_score": 5
          }
        ],
        "timeline": {
          "references": [
            {
              "type": "reference",
              "url": "http://bugs.python.org/issue13703",
              "source_type": "poc",
              "source": "nvd",
              "description": "",
              "published": "2012-10-05T05:00:00.000Z"
            }
          ],
          "repositories": []
        }
      }
    }
  ]
}